style_guides/accessibility_guide.md

```diff
@@ -0,0 +1,108 @@
+# Accessibility (A11Y) Guide
+
+This document provides some technical guidelines how to prevent several common
+accessibility issues.
+
+## Interactive elements
+
+### Use `<button>` and `<a>`
+
+**TL;DR** *Use `<button>` and `<a>` instead of click listeners on other elements
+and style it whatever way you need.*
+
+If you want to make an element clickable, use a `<button>` or `<a>` element for it.
+Use a *button* whenever it causes an action on the current page, and an *a* if it
+navigates to a different page. You can use click listeners just fine on these elements.
```

Comment:
I agree, but if I understand correctly, here are the scenarios we can have:

```html
<!-- Semantic -->
<a href="hardcoded/url">
<a href={dynamicUrl}>
<button onClick={doSomething}>
<button class="lookLikeALink" onClick={doSomething}>

<!-- Non-semantic -->
<a href onClick={doSomething}>
```

The last example is what I'm advocating against because it just doesn't make sense... you should just use a `<button>` element styled to look like a link. The href is supposed to provide a hint of where you're going, and the anchor tag should allow you to open the href in a new tab, but none of that functionality is being used.